barretenberg/cpp/src/barretenberg/client_ivc/mock_kernel_pinning.test.cpp

```diff
@@ -19,24 +19,36 @@ class MockKernelTest : public ::testing::Test {
 TEST_F(MockKernelTest, PinFoldingKernelSizes)
 {
     ClientIVC ivc;
-
+    ivc.precompute_folding_verification_keys();
     // Accumulate three circuits to generate two folding proofs for input to folding kernel
     GoblinUltraCircuitBuilder circuit_1{ ivc.goblin.op_queue };
     GoblinMockCircuits::construct_mock_function_circuit(circuit_1);
     ivc.initialize(circuit_1);
+    auto kernel_acc = std::make_shared<ClientIVC::VerifierInstance>();
```

Comment:
You're doing this in a bunch of places--could you add a constructor of a VerifierInstance from a verification key and use that?